src/refiners/foundationals/swin/mvanet/mcrm.py

```diff
@@ -0,0 +1,119 @@
+# Multi-View Complementary Refinement
+
+import torch
+from torch import Tensor, device as Device
+
+import refiners.fluxion.layers as fl
+
+from .utils import FeedForward, Interpolate, MultiPool, PatchMerge, PatchSplit, PatchwiseCrossAttention, Unflatten
+
+
+class Multiply(fl.Chain):
```

Comment:
Not to be confused with `src/refiners/fluxion/layers/basics.py`'s `Multiply` (nit: could be worth picking a different name - or just tackle later if/when we move it under fluxion utils)

Reply:
> I think we should keep it like this for now and discuss the name if we decide to put them in utils.

Agreed.